content/13.conclusion.md

```diff
@@ -1,13 +1,12 @@
+Beyond merely achieving good predictive performance in certain tasks, deep learning also has the potential to uncover hidden biological and clinical patterns of potentially high impact, which could fundamentally drive research discoveries and the delivery of new products to market.
+On the whole, deep learning must be approached with genuine thoughtfulness, caution, and responsibility if it is to reach its full potential.
 
-Through the tips and recommendations provided within this manuscript, we hope to encourage a prudent, vigilant community of computational practitioners, experimental biologists, and clinical scientists: colleagues who, before excitedly stitching together lines of code and datasets, first pause to think, dialogue, plan, and discern how their work might have far-reaching consequences with ethical dimensions.
-This holistic approach will help us to advance accountability, beneficence, and quality in science.
-
-Thus, we aim not only to increase the accessibility of deep learning techniques within the life sciences, but also to improve upon the reproducibility and interpretability of high-quality deep learning research in the literature and scientific community—especially given that published findings, models, and datasets will be leveraged to yield innovative tools, services, and products in the marketplace.
-Indeed, we hope that these tips will serve as a powerful engine for promoting meaningful discussions, reflections, team learnings, and best practices to drive collaboration that fosters cutting-edge deep learning innovation, sensibly and responsibly.
+Through the tips and recommendations provided within this manuscript, we have tried to increase the accessibility of deep learning techniques within the life sciences.  This is of critical importance given that published findings, models, and datasets will be leveraged to yield innovative tools, services, and products across societal spheres.
+We also hope to promote the development of a prudent and vigilant community of computational practitioners, experimental biologists, and clinical scientists that pauses to think, discuss, and carefully plan before excitedly stitching together lines of code that improperly apply deep (and tradition) learning techniques.
+After all, the initial discernment of the potential scientific and societal consequences of one's work is critical to the advancement of accountability, beneficence, and quality in science. By following the tips we have outlined and the lessons they underpin, such a community can promote deep learning research that is of high quality, interpretability, and reproducibility.
```

Comment:
These closing sentences are pretty words.  Can we close with a shorter, punchier take home message?

Reply:
I agree - I previously tried to condense the conclusion in this most recent edit, as it seemed to repeat a few core ideas, but I just took another swing at condensing it further.